Specs/Scene/ModelExperimental/ModelExperimentalSpec.js

```diff
@@ -1531,6 +1534,252 @@ describe(
+        expect(commands[0].pass).toBe(Pass.OPAQUE);
+        expect(commands[1].renderState.stencilTest.enabled).toBe(true);
+        expect(commands[1].pass).toBe(Pass.OPAQUE);
+
+        model.silhouetteColor = Color.fromAlpha(Color.GREEN, 0.0);
+        scene.renderForSpecs();
+        expect(commands.length).toBe(1);
+        expect(commands[0].renderState.stencilTest.enabled).toBe(false);
+        expect(commands[0].pass).toBe(Pass.OPAQUE);
+      });
+    });
+
+    it("silhouette works for invisible model", function () {
+      return loadAndZoomToModelExperimental(
+        {
+          gltf: boxTexturedGltfUrl,
+          silhouetteSize: 1.0,
+          color: Color.fromAlpha(Color.WHITE, 0.0),
+        },
+        scene
+      ).then(function (model) {
+        const commands = scene.frameState.commandList;
+        scene.renderForSpecs();
+        expect(commands.length).toBe(2);
+        expect(commands[0].renderState.colorMask).toEqual({
+          red: false,
+          green: false,
+          blue: false,
+          alpha: false,
+        });
+        expect(commands[0].renderState.depthMask).toEqual(false);
+        expect(commands[0].renderState.stencilTest.enabled).toBe(true);
+        expect(commands[0].pass).toBe(Pass.TRANSLUCENT);
+        expect(commands[1].renderState.stencilTest.enabled).toBe(true);
+        expect(commands[1].pass).toBe(Pass.TRANSLUCENT);
+      });
+    });
+
+    it("silhouette works for translucent model", function () {
+      return loadAndZoomToModelExperimental(
+        {
+          gltf: boxTexturedGltfUrl,
+          silhouetteSize: 1.0,
+          color: Color.fromAlpha(Color.WHITE, 0.5),
+        },
+        scene
+      ).then(function (model) {
+        const commands = scene.frameState.commandList;
+        scene.renderForSpecs();
+        expect(commands.length).toBe(2);
+
+        // Even though the silhouette color is opaque, the silhouette
+        // needs to be placed in the translucent pass.
+        expect(commands[0].renderState.stencilTest.enabled).toBe(true);
+        expect(commands[0].pass).toBe(Pass.TRANSLUCENT);
+        expect(commands[1].renderState.stencilTest.enabled).toBe(true);
+        expect(commands[1].pass).toBe(Pass.TRANSLUCENT);
+      });
+    });
+
+    it("silhouette works for translucent model and translucent silhouette color", function () {
+      return loadAndZoomToModelExperimental(
+        {
+          gltf: boxTexturedGltfUrl,
+          silhouetteSize: 1.0,
+          color: Color.fromAlpha(Color.WHITE, 0.5),
+          silhouetteColor: Color.fromAlpha(Color.RED, 0.5),
+        },
+        scene
+      ).then(function (model) {
+        const commands = scene.frameState.commandList;
+        scene.renderForSpecs();
+        expect(commands.length).toBe(2);
+
+        // Even though the silhouette color is opaque, the silhouette
+        // needs to be placed in the translucent pass.
```

Comment:
This might have been true of the previous test, but in this one the silhouette color is translucent.